ci-operator/step-registry/mco/conf/day1/tls-security-profile/mco-conf-day1-tls-security-profile-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: mco-conf-day1-tls-security-profile
+  from_image:
+    namespace: origin
+    name: centos
+    tag: '8'
+  commands: mco-conf-day1-tls-security-profile-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: MCO_CONF_DAY1_TLS_PROFILE
+      default: '{"type": "Custom", "custom": { "ciphers": ["ECDHE-ECDSA-CHACHA20-POLY1305", "ECDHE-RSA-CHACHA20-POLY1305", "ECDHE-RSA-AES128-GCM-SHA256", "ECDHE-ECDSA-AES128-GCM-SHA256"], "minTLSVersion": "VersionTLS11"}}'
+      documentation: |
+        Name of the tlsSecurityProfile that will be set for cluster after installation is complete
```

Comment:
"Name of the tlsSecurityProfile"

It is not the name of the tlsSecurityProfile, but the tlsSecurityProfile configuration